cypress/integration/articleFlows/mangageFlows/lockDiscussion.spec.js

```diff
@@ -0,0 +1,107 @@
+describe('Lock discussion', () => {
```

Comment:
it's tiny thing but there's a typo in the subdirectory - `mangageFlows` -> `manageFlows`